bin/roma-adm

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby
+require 'pathname'
+require 'optparse'
+
+base_path = Pathname(__FILE__).dirname.parent.expand_path
+$LOAD_PATH.unshift("#{base_path}/lib")
+
+require 'roma/tools/roma-adm'
+
+# opt parse
+opts = OptionParser.new
+opts.banner="usage:#{File.basename($0)} [command] [port No.]"
+opts.on("-h", "--help", "Show this message") {
+  puts opts; exit
+}
+opts.parse!(ARGV)
```

Comment:
You think it is better to cover exception not only main logic but also all logic including optpaser and argument check, right?
Now usage has a just few info, so I will improve it.